test/TestHelper.cpp

```diff
@@ -352,29 +352,72 @@ void ImportTest::exportTest(bytes const& _output, State const& _statePost)
 	m_TestObject["transaction"] = makeAllFieldsHex(m_TestObject["transaction"].get_obj());
 }
 
+std::string jsonHash(h256 const& _value) { return toString(_value); }
+std::string jsonHash(Nonce const& _value) { return toString(_value); }
+std::string jsonHash(LogBloom const& _value) { return toString(_value); }
+std::string jsonHash(Address const& _value)	{ return toString(_value);	}
+std::string jsonHex(bytesConstRef _code)	{ return "0x" + toHex(_code);	}
+std::string jsonHex(bytes const& _code)	{ return "0x" + toHex(_code);	}
```

Comment:
those functions have nothing to do with json (no input or output parameter is a json value). I don't think we need them (except of the last one). We can use toString or toHex directly or use your helper function makeAllFieldsHex.